dev/devicelab/lib/tasks/plugin_tests.dart

```diff
@@ -370,6 +370,7 @@ public class $pluginClass: NSObject, FlutterPlugin {
           destination: 'platform=macOS',
           configuration: 'Debug',
           testName: 'native_plugin_unit_tests_macos',
+          extraOptions: <String>['-parallel-testing-enabled', 'NO'],
```

Comment:
This is potential fix for https://github.com/flutter/flutter/issues/164902:

Tests indicate it seem to be effective:
https://ci.chromium.org/ui/p/flutter/builders/try/Mac%20plugin_test_macos/27007/overview
https://ci.chromium.org/ui/p/flutter/builders/try.shadow/Mac%20plugin_test_macos/7/overview
https://ci.chromium.org/ui/p/flutter/builders/try.shadow/Mac%20plugin_test_macos/6/overview
https://ci.chromium.org/ui/p/flutter/builders/try.shadow/Mac%20plugin_test_macos/8/overview